commands/channel.go

```diff
@@ -76,6 +76,16 @@ Archived channels are appended with ' (archived)'.`,
 	RunE:    withClient(listChannelsCmdF),
 }
 
+var ModifyChannelCmd = &cobra.Command{
+	Use:   "modify [channel] [flags]",
+	Short: "Modify a channel's public/private type",
+	Long: `Change the public/private type of a channel.
+Channel can be specified by [team]:[channel]. ie. myteam:mychannel or by channel ID.`,
+	Example: "  channel modify myteam:mychannel --private",
+	Args:    cobra.MinimumNArgs(1),
```

Comment:
I think this can be changed to:

```suggestion
	Args:    cobra.ExactArgs(1),
```